modules/hcp-ibmz-adding-credentials-registry.adoc

```diff
@@ -0,0 +1,61 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-adding-credentials-registry_{context}"]
+= Adding Credentials and Registry Certificate Authority to the Management Cluster
+
+To pull the mirror registry images from the Management Cluster , you must first add Credentials and Registry Certificate Authority of mirror registry to the Management cluster. Follow the procedure below:
+
```

Comment:
Before the procedure, add the `.Procedure` heading. See the Procedure template in the modular documentation guidelines for an example: https://github.com/redhat-documentation/modular-docs/blob/main/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc?plain=1